install/helm/agones/templates/crds/k8s/_io.k8s.api.core.v1.PodTemplateSpec.yaml

```diff
@@ -200,7 +200,9 @@ properties:
                                 description: The label key that the selector applies to.
                                 type: string
                               operator:
-                                description: Represents a key's relationship to a set of values. Valid operators are In, NotIn, Exists, DoesNotExist. Gt, and Lt.
+                                description: |+
```

Comment:
I love that this hacky bash script is still working.

Reply:
😃